autoload/elixir/indent.vim

```diff
@@ -22,6 +22,7 @@ let s:PAIR_START = '\<\%('.s:NO_COLON_BEFORE.s:BLOCK_START.'\)\>'.s:NO_COLON_AFT
 let s:PAIR_MIDDLE = '^\s*\%('.s:BLOCK_MIDDLE.'\)\>'.s:NO_COLON_AFTER.'\zs'
 let s:PAIR_END = '\<\%('.s:NO_COLON_BEFORE.s:BLOCK_END.'\)\>\zs'
 let s:LINE_COMMENT = '^\s*#'
+let s:MATCH_OPERATOR = '[^!><=]=[^~=>]'
```

Comment:
Technically it's the match operator I believe, but I'm up for calling it anything - let me know which you'd prefer http://elixir-lang.org/getting-started/pattern-matching.html#the-match-operator